ProjectSettings/Packages/com.unity.testtools.codecoverage/Settings.json

```diff
@@ -6,7 +6,7 @@
             {
                 "type": "System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
                 "key": "IncludeAssemblies",
-                "value": "{\"m_Value\":\"com.playeveryware.eos,com.playeveryware.eos.core,com.playeveryware.eos.samples,com.playeveryware.eos.samples.appleauth,com.playeveryware.eos.samples.appleauth.editor,com.playeveryware.eos.samples.core,com.playeveryware.eos.samples.facepunch-steamworks,com.playeveryware.eos.samples.networking,com.playeveryware.eos.samples.oculus,com.playeveryware.eos.samples.postprocessing,com.playeveryware.eos.samples.steam,com.playeveryware.eos-Editor,com.playeveryware.eos-Editor.steam.utility\"}"
+                "value": "{\"m_Value\":\"com.playeveryware.eos,com.playeveryware.eos.core,com.playeveryware.eos-Editor,com.playeveryware.eos-Editor.steam.utility\"}"
```

Comment:
Should we have code coverage for the samples namespace? I was making tests for the Manager/Services.

Reply:
Eventually we should, but the priority is the non-sample code.